scripts/tools/zap/run_zaptool.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+CHIP_ROOT=$(readlink -f ${BASH_SOURCE%/scripts/tools/zap/run_zaptool.sh})
+ZAP_ROOT="$CHIP_ROOT/third_party/zap/repo"
+[[ -n "$1" ]] && ZAP_ARGS=(-i "$(readlink -f "$1")") || ZAP_ARGS=()
+
+
+cd $CHIP_ROOT \
+&& git submodule update --init third_party/zap/repo \
```

Comment:
pedantic: I would indent the lines starting with "&&" or "||" (to be more consistent with existing scripts).